test/ReactSetStateTestComponent.js

```diff
@@ -0,0 +1,88 @@
+function getUpdatingSetStateLifeCycleCalls() {
```

Comment:
Yeah this is pretty much to make sure that they are the same. We could do a static list but then if ReactJs ever changes we would not know about it.